package.json

```diff
@@ -33,7 +33,8 @@
     "test": "jest --verbose false --runInBand --force-exit",
     "start": "nodemon server/server.js",
     "react-dev": "webpack -d --watch",
-    "seed": "node database/seedScript.js"
+    "seed": "database/seedScript.js",
```

Comment:
Def think you'd be best served by seeding your db in your start script instead of doing it in its own script 'seed'. I know you're working on it!!!